artifactory/services/security.go

```diff
@@ -23,10 +27,42 @@ func NewSecurityService(client *rthttpclient.ArtifactoryHttpClient) *SecuritySer
 	return &SecurityService{client: client}
 }
 
+// GetAPIKeyPath return the apiKeyPath
```

Comment:
Since *apiKeyPath* is a const, I think we can remove this function and make *apiKeyPath* public (**ApiKeyPath**).